FrameMetadata/Lets-Build-OTPublisher/AppDelegate.m

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.m
+//  Lets-Build-OTPublisher
+//
+//  Copyright (c) 2013 TokBox, Inc. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+@implementation AppDelegate
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    // Override point for customization after application launch.
+    return YES;
+}
+							
+- (void)applicationWillResignActive:(UIApplication *)application
+{
+    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
```

Comment:
I usually like to remove all these comments since they are automatically placed by xcode

Reply:
We will have to do a batch update on all projects across this repo then.

Reply:
Yep, we should do it in the rest of the samples, but let's start with this one